src/driver/amdxdna/ve2_of.c

```diff
@@ -3,8 +3,71 @@
  * Copyright (C) 2025, Advanced Micro Devices, Inc.
  */
 
+#include <linux/xlnx-ai-engine.h>
+#include <linux/firmware.h>
+
 #include "ve2_of.h"
 
+static int ve2_load_cert(struct amdxdna_dev_hdl *ndev)
```

Comment:
In ve2_init(), the variable was name "xdna_hdl". But now it is "ndev". Please keep the naming consistent.